modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -376,7 +376,7 @@
     ::upload/varchar-255              [[:varchar 255]]
     ::upload/text                     [:text]
     ::upload/int                      [:bigint]
-    ::upload/auto-incrementing-int-pk [:bigint [:identity 0 1]]
+    ::upload/auto-incrementing-int-pk [:bigint [:identity 1 1]]
```

Comment:
Why did this need to change? It would be nice if a comment explained these numeric params.

Reply:
Are you still planning to add this comment?

Reply:
done d85d308f0371d9d532c7be75ab2e31573a1b3e05